test/unit/services/platform-test.js

```diff
@@ -343,6 +343,23 @@ describe('Platform', function() {
     });
   });
 
+  describe('getFreeDiskSpace', function() {
+    describe('on windows', function() {
+
+      beforeEach(function() {
+        sandbox.stub(Platform, 'getOS').returns('win32');
+      });
+
+      it('should able to return free disk space', function() {
+        sandbox.stub(child_process, 'exec').yields(undefined, '248005160960');
+        let location = 'C:\\DevelopmentSuite';
+        return Platform.getFreeDiskSpace(location).then((result) => {
+          expect(result).to.be.equal(248005160960);
+        });
+      });
+    });
```

Comment:
It also should test command actually parsing location passed as a parameter and puts correct disk name in to executed command.